pkg/kritis/crd/securitypolicy/securitypolicy.go

```diff
@@ -51,9 +52,26 @@ func ImageSecurityPolicies(namespace string) ([]v1beta1.ImageSecurityPolicy, err
 	return list.Items, nil
 }
 
+type Validator struct {
+	ImageToVulnz map[string][]metadata.Vulnerability
```

Comment:
Add comment for Public types.
Lets make the cache private.

Reply:
done